plutus-metatheory/src/Builtin.lagda

```diff
@@ -22,7 +22,10 @@ data Builtin : Set where
   verifySignature          : Builtin
   equalsByteString         : Builtin
   ifThenElse               : Builtin
+  charToString             : Builtin
+  append                   : Builtin
+  trace                    : Builtin
```

Comment:
@jmchapman @michaelpj I added these three builtins that we previously considered "dynamic" and didn't care about.